std/algorithm/iteration.d

```diff
@@ -4611,12 +4611,12 @@ if (isSomeChar!C)
 /**
 Sums elements of $(D r), which must be a finite
 $(REF_ALTTEXT input range, isInputRange, std,range,primitives). Although
-conceptually $(D sum(r)) is equivalent to $(LREF fold)!((a, b) => a +
-b)(r, 0), $(D sum) uses specialized algorithms to maximize accuracy,
+conceptually $(D sum(r)) is equivalent to $(LREF fold)$(D !((a, b) => a +
+b)(r, 0)), $(D sum) uses specialized algorithms to maximize accuracy,
 as follows.
 
 $(UL
-$(LI If $(D $(REF ElementType, std,range,primitives)!R) is a floating-point
+$(LI If $(REF ElementType, std,range,primitives)$(D !R) is a floating-point
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4370550/23589559/88d2cb2e-01cf-11e7-8313-5717ffd78c9c.png)

NG: this was broken before and is now fixed:
http://dlang.org/phobos/std_algorithm_iteration.html#.sum

![image](https://cloud.githubusercontent.com/assets/4370550/23589685/e9287e2c-01d1-11e7-91aa-ddf4198cf52a.png)